.github/workflows/compilemmd.yml

```
@@ -0,0 +1,42 @@
+name: Node.js CI
+
+on:
+  push:
+    paths:
+      - "**/*.mermaid"
+      - "**/*.mmd"
+      - "**/*.md"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js 16
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
+
+      - name: Install mermaid-cli
+        run: npm i
```

[want] yarn 使いたいですねえ
```suggestion
      - name: Check for Yarn and Install if Needed
        run: yarn -v || npm install -g yarn

      - name: Install dependencies
        run: yarn install
```